app/models/user.rb

```diff
@@ -58,6 +58,25 @@ class User < ActiveRecord::Base
   scope :last_signed_in_after, lambda { |date| web_users.not_suspended.where('date(current_sign_in_at) >= date(?)', date) }
   scope :not_recently_unsuspended, lambda { where(['unsuspended_at IS NULL OR unsuspended_at < ?', UNSUSPENSION_GRACE_PERIOD.ago]) }
 
+  scope :with_status, lambda { |status|
+    case status
```

Comment:
if a new status is added (eg for 2sv) and we forget to update this code (which is quite likely), this statement will not filter correctly. Is it worth adding a final condition that sends an exception to Errbit when the status hasn't been matched?